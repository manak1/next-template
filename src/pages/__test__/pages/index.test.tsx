import React from 'react'
import HomePage from 'src/pages/index'

test('ホーム画面', () => {
  const home = <HomePage />
  expect(home).toMatchSnapshot()
})
