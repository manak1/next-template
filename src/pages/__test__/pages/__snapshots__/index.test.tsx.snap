// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ホーム画面 1`] = `<Home />`;
